lib/lure/pr.go

```diff
@@ -103,15 +101,22 @@ func getPullRequests(auth Authentication, username string, repoSlug string) []Pu
 
 	return list.PullRequest
 }
-func getPRRequest(prRequest *http.Request) (*http.Response, error) {
-	resp, e := http.DefaultClient.Do(prRequest)
-	for !(resp.StatusCode == 200 && resp.StatusCode < 300) {
-		log.Printf("Getting '%s' PR returned %d. Retrying...", prRequest.URL, resp.StatusCode)
-		resp, e = http.DefaultClient.Do(prRequest)
-		time.Sleep(time.Second)
+func getPRRequest(prRequest *http.Request) (*PullRequestList, error) {
+	client := getHTTPClient()
+	resp, err := client.Do(prRequest)
+
+	if err != nil {
+		log.Println("Error getting PR Requests", client.LogString())
```

Comment:
what's considered an error? e.g. you don' tneed to check status code anymore. or there's no point because we'll just fail to parse it anyway?

Reply:
Well I think we still have to check status code. The retryer allow us to be more resilient to temporary errors, but there are still cases where after all the retries we'll still fail to fetch the PR list.

As you mentioned, at this point I'd rather signify that there was an error instead of returning an empty list. The caller can make the choice to stop or continue from that point on.